core/src/main/scala/cats/data/WriterT.scala

```diff
@@ -184,14 +184,21 @@ private[data] sealed abstract class WriterTInstances8 extends WriterTInstances9
   }
 }
 
-private[data] sealed abstract class WriterTInstances9 {
+private[data] sealed abstract class WriterTInstances9 extends WriterTInstances10 {
   implicit def catsDataMonadErrorForWriterT[F[_], L, E](implicit F: MonadError[F, E], L: Monoid[L]): MonadError[WriterT[F, L, ?], E] =
     new WriterTMonadError[F, L, E] {
       implicit val F0: MonadError[F, E] = F
       implicit val L0: Monoid[L] = L
     }
 }
 
+private[data] sealed abstract class WriterTInstances10 {
+  implicit def catsDataFunctorForWriterT[F[_], L](implicit F: Functor[F]): Functor[WriterT[F, L, ?]] =
```

Comment:
Not sure why this is needed. The [`Coflatmap` instance](https://github.com/edmundnoble/cats/blob/1f460fab573ad4150885743bfbd5e2f1b7eea466/core/src/main/scala/cats/data/WriterT.scala#L177-L180) should surfice, no?